extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcIdentityProvider.java

```diff
@@ -35,6 +36,11 @@
 
 @ApplicationScoped
 public class OidcIdentityProvider implements IdentityProvider<TokenAuthenticationRequest> {
+
+    @SuppressWarnings("deprecation")
+    private static final Uni<AccessToken> NULL_CODE_ACCESS_TOKEN_UNI = Uni.createFrom().nullItem();
```

Comment:
Why removing the deprecation?

Reply:
@cescoffier What do you mean ? `AccessToken` is deprecated in Vert.x 3.8.x hence I add to avoid the IDE warnings...

Reply:
ah ok, I was referring to the mutiny part.